package.json

```diff
@@ -116,8 +116,8 @@
   "devDependencies": {
     "@kadira/storybook": "^2.35.3",
     "babel-eslint": "^7.2.3",
-    "chai": "^3.5.0",
-    "chai-enzyme": "^0.6.1",
+    "chai": "^4.0.1",
```

Comment:
https://github.com/chaijs/chai/releases/tag/4.0.0

> ### Breaking Changes
>
> We no longer support Node v0.10 and v0.12 (since their LTS has ended) _(PRs: [#816](https://github.com/chaijs/chai/pull/816), [#901](https://github.com/chaijs/chai/pull/901))_
>
> ...

Other changes are unrelated changes to Mastodon.